test/de/frosner/broccoli/services/NomadServiceSpec.scala

```diff
@@ -0,0 +1,84 @@
+package de.frosner.broccoli.services
+
+import java.util.concurrent.TimeUnit
+
+import de.frosner.broccoli.controllers.ServiceMocks
+import de.frosner.broccoli.conf
+import de.frosner.broccoli.nomad.models.{Job, Service, Task, TaskGroup}
+import org.specs2.mutable.Specification
+import org.mockito.Mockito._
+import play.api.Configuration
+import play.api.libs.json._
+import play.api.mvc._
+import play.api.routing.sird._
+import play.api.test._
+import play.core.server.Server
+
+import Job.jobFormat
+
+import scala.concurrent.Await
+import scala.concurrent.duration.Duration
+
+class NomadServiceSpec extends Specification with ServiceMocks {
+
+  sequential
+
+  "Requesting services for specific job" should {
+
+    "ask consul for the services that nomad returns" in {
```

Comment:
Please write test descriptions based on the intended outcome not the intended procedure, eg,

```
"The Nomad service" should "provide services for a job with running consul services"

"The Nomad service" should "provide services for a job which has tasks without services"
```